lib/asciidoctor-pdf/prawn_ext/extensions.rb

```diff
@@ -862,6 +862,10 @@ def keep_together_if verdict, &block
     end
   end
 
+  def play_icon
```

Comment:
I'd rather keep this inline rather than adding a method (even though it's duplicated once).

Reply:
Sure!